salt/modules/win_pkg.py

```diff
@@ -650,7 +663,8 @@ def refresh_db(**kwargs):
     __salt__['cp.cache_dir'](
```

Comment:
Before this line can be add
```
if repo_details.winrepo_source_dir starts with salt://
  path_without_salt=remove salt://
  log.info('Fetching *.sls files for saltenv \'%s\'  path \'%s\' i.e. master\'s file_roots \'%s\' under \'%s\'', saltenv, repo_details.winrepo_source_dir, saltenv, path_without_salt)
else
  log.info('Fetching *.sls files from  \'%s\'', repo_details.winrepo_source_dir)
```

Might help reduce the number of questions we get.

Reply:
Good idea to add at least something that logs what we're doing. I opted to just output the `repo_details.winrepo_source_dir` here.